addons/website/static/src/scss/website.scss

```diff
@@ -196,6 +196,17 @@ $-seen-urls: ();
     }
 }
 
+// We don't want the area to be visible in edit mode when we edit the popup that
+// is shared on all pages, otherwise the area takes the height of the cursor to
+// type text.
+#o_shared_blocks {
+    font-size: 0px !important;
+
+    * {
+        font-size: initial;
+    }
+}
```

Comment:
I don't think this is very consistent to make a hack to hide the cursor in that "share_blocks" area and at the same time suggest it could later contain something else than popups (since its name). I don't think you tested this much either: since you force "font-size: initial" on **all elements** inside, you are basically breaking the ability to change font-size in the popup:
- Add a popup shared on all pages
- Set the font-size of a sentence inside to 10px -> it works as inline style on the element
- Now set one word of that sentence in bold -> that word will go back to default font size... since your rule applies on the newly created element.

Maybe could be fixed by setting `> *` as you don't need to target all elements but only children... but we should avoid this hack altogether.

Recently @rdeodoo [made a fix](https://github.com/odoo/odoo/commit/3da2835b93dbb398e4abe217fe721e693b8925c2) in stable that makes the `.s_popup` element hidden when the `.modal` inside is hidden. I think we should do something similar here and more:
- I don't want `#o_shared_blocks` to be visible (even if empty) if what it contains is only an hidden modal
- When the modal is opened, the `#o_share_blocks` element should be made visible first, just like we do it for the `.s_popup` element. In that case, and when there is only one modal inside of this, you could maybe just force `height: 0` on this element to avoid the cursor problem? If `height: 0;` is not enough, maybe `user-select` would be better than the font-size trick?